.eslintrc.json

```diff
@@ -20,7 +23,11 @@
     ".prettierignore",
     "*.json",
     "public/*",
-    "pretter.config.js"
+    "*.test.js",
+    "*.test.ts",
+    "pretter.config.js",
+    "jest.config.mjs",
+    "jest.setup.ts"
```

Comment:
why are we adding those to eslint? can we not run prettier on them?

Reply:
I believe I added them because my local esint parser was giving me the following warning that eslint is parsing files not in the path of tsconfig:

`Parsing error: ESLint was configured to run on `<tsconfigRootDir>/jest.config.mjs` using `parserOptions.project`: /nycmesh-node-visualizer/tsconfig.json
However, that TSConfig does not include this file. Either:
- Change ESLint's list of included files to not include this file
- Change that TSConfig to include this file
- Create a new TSConfig that includes this file and include it in your parserOptions.project
See the typescript-eslint docs for more info: https://typescript-eslint.io/linting/troubleshooting#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file`

Should I undo the eslintrc changes?

Reply:
Ah okay, this seems fine then. Was just curious on why they are there.